demos/storybook/storybook/stories/bucket-view.tsx

```diff
@@ -59,7 +59,7 @@ storiesOf('BucketView')
                     title={device.name}
                     subtitle={`aquired: ${device.dateAcquired.toUTCString()}`}
                     backgroundColor={white[100]}
-                    color={device.status === 'stopped' ? red[800] : blue[800]}
+                    statusColor={device.status === 'stopped' ? red[800] : blue[800]}
```

Comment:
Is this a separate bug?

Reply:
I came across that when I was doing some find and replace stuff. It's in a file that's not currently used (we haven't innersourced the bucketview yet), so it's just silly. But it was using an outdated API.